docs: versioning docs should show example versioned sidebar files (#9500)
facebook/docusaurus: website/docs/guides/docs/versioning.mdx

</Tabs>
```

:::tip

> Versioned sidebar files are, like standard sidebar files plugin root relative - so for the example above your versioned sidebar file may look like:

Reviewer: ```suggestion
Versioned sidebar files are, like standard sidebar files, relative to the content root for the given version — so for the example above, your versioned sidebar file may look like:
```

(Docs plugins use both `/docs` and `/versioned_docs` so there's not a single root to be "relative to".)

> ```js

Reviewer: ````suggestion
```json
````

...versioned sidebars are JSON, right? (You can see how rusty I am already.)

{
  "sidebar": [
    {
      "type": "autogenerated",

>       "dirName": "."

Reviewer: You can have autogenerated versioned sidebars?! Does the versioning CLI produce this now?

Author: I don't believe the CLI produces these but they do certainly appear to work.

Reviewer 2: > Does the versioning CLI produce this now?

yes we don't substitute autogenerated items anymore for a while 👍 

![CleanShot 2023-11-10 at 18 47 07@2x](https://github.com/facebook/docusaurus/assets/749374/92356093-bd15-41e4-a73b-819d52f42929)

    }
  ]
}
```

or for a manual sidebar:

> ```js

Reviewer: Same here?

````suggestion
```json
````
